spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/ConsumeOkEvent.java

```diff
@@ -20,6 +20,8 @@
 
 /**
  * @author Gary Russell
```

Comment:
Hmmm - shouldn't checkstyle bark at no description?

Reply:
I think we don't have it enabled too many classes are without JavaDocs.

```
<module name="JavadocStyle">
	<property name="checkEmptyJavadoc" value="false"/>
</module>
```
That's what we have in the `checkstyle.xml`

Reply:
ok; yeah; I guess we have it in the newer projects.